source/common/http/filter_manager.cc

```diff
@@ -1288,6 +1288,8 @@ absl::optional<Router::ConfigConstSharedPtr> ActiveStreamDecoderFilter::routeCon
   return parent_.filter_manager_callbacks_.routeConfig();
 }
 
+bool ActiveStreamEncoderFilter::canContinue() { return !parent_.state_.encoding_complete_; }
```

Comment:
can we comment somewhere why we use local complete on the decoder path  and encoding complete on the encoder path?